src/base/torrentfileguard.cpp

```diff
@@ -31,6 +31,11 @@
 #include "settingvalue.h"
 #include "utils/fs.h"
 
+namespace
+{
+    SettingValue<TorrentFileGuard::AutoDeleteMode> autoDeleteModeSetting(u"Core/AutoDeleteAddedTorrentFile"_qs);
```

Comment:
I doubt that using this variable is any better than what it was before.

If you just want to have the name of the setting in single place, then the function here looked preferable.

Reply:
PR updated, I hope I understand you correctly.

Reply:
> PR updated, I hope I understand you correctly.

Yes.